### PR TITLE
Fix all checkstyle errors

### PR DESCRIPTION
**Pull request by benjamin-confino**
This fixes all checkstyle errors in the repository

I won't blame you if you want to hold off on this until after https://github.com/eclipse/microprofile-parent/pull/82/files is merged and published to avoid the pom.xml changes.

### CODE REVIEW

**Comment by benjamin-confino** [comment]
I've been informed that Microparent will be updated soon, so I'll deliver this later with new version to avoid all that TCK stuff.

File: pom.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
     <parent>
         <groupId>org.eclipse.microprofile</groupId>
         <artifactId>microprofile-parent</artifactId>
-        <version>2.8</version>
+        <version>2.9-RC1</version>
```
Comment (Azquelt) [inline pom.xml:22]: We would need to update this before release.

**Review by Azquelt — APPROVED** [review]
I think these changes are fine, as long as we switch to a released version of the microprofile-parent artifact before release. (Either a released 2.9 or back to 2.8 and accept the checkstyle warnings).

File: pom.xml (hunk 1 above)
Comment (Azquelt) [reply pom.xml:22]: Opened #623 so we don't forget to do the update.

**Pull request merged**